tests/Extension/RedirectedURLHandlerTest.yml

```diff
@@ -4,11 +4,11 @@ SilverStripe\RedirectedURLs\Model\RedirectedURL:
     To: /site-info/log-in/
   redirect-with-query:
     FromBase: /query-test-with-query-string
-    FromQueryString: foo=bar
+    FromQuerystring: foo=bar
```

Comment:
I'd keep it as FromQueryString as 'query string' are two words.

Reply:
I do agree that it should be "query string", but I think that this would be a breaking change that we don't need. The field is defined as `FromQuerystring`. This test was using an incorrect field name, and causing the test to break. All I was doing here was updating the fixture to use the correct name.

https://github.com/silverstripe/silverstripe-redirectedurls/blob/master/src/Model/RedirectedURL.php#L46

Reply:
Since it will be a new major version it doesn't matter?

Reply:
But it might cause issues with data migration, so probably leave it as is. Who knows what different db engines would do with the same field name with a different case in it. Not worth the hassle.

Reply:
@michalkleiner I mean, it matters insomuch as we'll all lose our existing redirects when we upgrade, and I'll have to provide a dev/task (or something) to migrate all of these field values over to the new field.

I would prefer to keep the upgrade path easy.